README.md

```diff
@@ -1,83 +1,120 @@
+        <img src="https://img.shields.io/github/commit-activity/m/filecoin-project/bacalhau" />
+        </a>
+    <a href="https://img.shields.io/github/downloads/filecoin-project/bacalhau/total">
+        <img src="https://img.shields.io/github/downloads/filecoin-project/bacalhau/total" alt="total download">
+        </a>
+     <a href="https://github.com/filecoin-project/bacalhau/graphs/contributors">
+    <img src="https://img.shields.io/github/contributors/filecoin-project/bacalhau" alt="Bacalhau contributors" >
+    </a>
+    <a href="https://www.bacalhau.org/">
+    <img alt="Bacalhau website" src="https://img.shields.io/badge/website-bacalhau.org-red">
   </a>
+      <a href="https://filecoinproject.slack.com/" alt="Slack">
+        <img src="https://img.shields.io/badge/slack-join_community-red.svg?color=0052FF&labelColor=090422&logo=slack" />
+        </a>
+    <a href="https://twitter.com/intent/follow?screen_name=BacalhauProject">
+        <img src="https://img.shields.io/twitter/follow/BacalhauProject?style=social&logo=twitter" alt="follow on Twitter">
+        </a>
 </p>
 
+[Bacalhau](https://www.bacalhau.org/) is a platform for public, transparent, and optionally verifiable distributed computation that helps you manage your parallel processing jobs. Bacalhau enables users to run arbitrary docker containers and wasm images as tasks against data stored in IPFS. This architecture is referred to as Compute Over Data (CoD). Bacalhau was coined from the Portuguese word for salted Cod fish. 
 
-Learn more about the project from our [Website](https://www.bacalhau.org/), [Twitter](https://twitter.com/BacalhauProject) & [YouTube Channel](https://www.youtube.com/channel/UC45IQagLzNR3wdNCUn4vi0A).
+## Table of Contents
+- [Features](#features)
+- [Getting started](#getting-started)
+  - [Learn more](#learn-more)
+- [Documentation](#documentation)
+- [Developers guide](#developers-guide)
+  - [Running Bacalhau locally](#running-bacalhau-locally)
+  - [Releasing a new version](#releasing-a-new-version)
+  - [Notes for Dev contributors](#notes-for-dev-contributors)
+- [Ways to contribute ](#ways-to-contribute)
+- [Current state of Bacalhau](current-state-of-bacalhau)
+- [License](#license)
 
-## Latest Updates
+## Features
+- **Process jobs fast**: Jobs in Bacalhau are processed where the data was created (meaning no ingress/egress) and all jobs are parallel by default.
+- **Low cost:** Bacalhau uses the compute that produced the data in the first place and reuses the existing hardware you already have. You also save on any ingress/egress fees you may have been charged. The cost of computing is much cheaper than managed services.
+- **Secure**: Data is not collected in a central location before processing, meaning all scrubbing and security can be applied at the point of collection.
+- **Large-scale data**: With Bacalhau, you can manage your own containers that can scale to batch process petabytes (quadrillion bytes) files.
```

Comment:
> you can manage your own containers

What management are we referring to? I’d rather be running processing without managing stuff!